### DISCUSSION TITLE
CascadeStudio vs OpenScad and round-anything Library

### DISCUSSION THREAD

**Discussion by Irev-Dev** [zalo/CascadeStudio#50] [Show and tell]
Not sure if I should laugh or cry about this, but basically I wrote an entire library to be able to add fillets to arbitrary polygons in OpenSCAD and CascadeStudio has the same functionality out of the box.

![vsOpenScad](https://user-images.githubusercontent.com/29681384/101981160-f530c900-3cbe-11eb-8f4e-23446f826759.jpg)

My library is [here](https://github.com/Irev-Dev/Round-Anything), and it's even featured on [openscad.org](http://www.openscad.org/libraries.html#Round%20Anything), but I would recommend just using CascadeStudio instead 😅

**Comment by zalo** [comment]
Your library is awesome, and taps into the deep and rich OpenSCAD culture and ecosystem (one that I'm proud to say that I came from).

CascadeStudio is largely an idea for what OpenSCAD can be in the modern age, but I doubt CascadeStudio (or its descendants) will ever have the reach/impact that OpenSCAD has, and that's why I think your library is still totally cool and relevant.

CAD Kernels like OpenCASCADE can do a few things (like Spherical Minkowski Sums, Fillets, and Lofts) really nicely, but they also lack some of the key and beautiful functionality that makes OpenSCAD so impactful (Convex Hulls, Non-Spherical Minkowski Sums, and native STL/Mesh Editing).  For that reason, I think there will always be a place for OpenSCAD, and libraries that improve its functionality.

**Reply by Irev-Dev** [reply]
OpenSCAD definitely does have a nice language from the perspective of being very easy to hit the ground running with it.
